slidez-core/src/main/java/com/binarystudio/academy/slidez/app/user/UserController.java

```diff
@@ -52,4 +48,14 @@
 		return new ResponseEntity<>(userDetailsDto, HttpStatus.OK);
 	}
 
+	@PutMapping("{id}")
+    public ResponseEntity<Object> updateUser(@PathVariable("id") UUID id,  User user) {
```

Comment:
ResponseEntity with Object generic is ambiguous. I want you to either create Dto which transfers UserDetailsDto and status, or return UserDetailsDto only in case of success, and nothing otherwise. From hardcoded strings will suffer frontend - because we will have to check, what did backend return before proceed, and it will be harder to implement internationalization